cardano-db-sync/src/Cardano/DbSync/Metrics.hs

```diff
@@ -34,9 +57,8 @@ withMetricsServer port action = do
 makeMetrics :: RegistryT IO Metrics
 makeMetrics =
   Metrics
-    <$> registerGauge "db_block_height" mempty
-    <*> registerGauge "remote_tip_height" mempty
-    <*> registerGauge "action_queue_length_pre" mempty
-    <*> registerGauge "action_queue_length_post" mempty
-    <*> registerGauge "action_queue_length_post_write" mempty
+    <$> registerGauge "remote_block_tip_height" mempty
+    <*> registerGauge "database_queue_height" mempty
+    <*> registerGauge "local_block_number" mempty
+    <*> registerGauge "local_slot_number" mempty
```

Comment:
Have you socialised these name changes for existing metrics? Are @iohk-devops aware of them?

Reply:
There is a comment for notifying them. These names are much clearer.

Reply:
I think `remote_block_tip_height` could be rather `remote_tip_block_height`.

But more importantly I'm not sure what would be the new name of `db_block_height`, is it `database_queue_height` or `local_block_number`?

In either case I think the original name was clearer.

FYI, in our alerting we compare `db_block_height` with `cardano_node_metrics_blockNum_int`.

Also what about using similar convention as `cardano-node` ? something like
`cardano_db_sync_DbBlockNum_int` (the `metrics` part is useless, but the application prefix, especially, is useful).

Reply:
> I think remote_block_tip_height could be rather remote_tip_block_height.

Fine, will fix.

> But more importantly I'm not sure what would be the new name of db_block_height, is it database_queue_height or local_block_number?

`db_block_height` is the height of blocks in the database. So the `local_block_number` is the information from the database and `database_queue_height` is the number of `DbAction` in the queue waiting for DB submission (this can be used to track how well the DB is performing and whether there is some congestion).

`db_block_height` it is, then.

The last part changes things. To just to see whether we understand each other, new names would be:
- `remote_block_tip_height` - `cardano_db_sync_node_block_height`
- `database_queue_height` - `cardano_db_sync_db_queue_height`
- `local_block_number` - `cardano_db_sync_db_block_height`
- `local_slot_number` - `cardano_db_sync_db_slot_height`

Does this look ok, @jbgi ?

Reply:
yep, lgtm! thanks.